nilearn/signal.py

```diff
@@ -120,6 +120,38 @@ def _mean_of_squares(signals, n_batches=20):
     return var
 
 
+def _sum_of_squares(signals, n_batches=20):
```

Comment:
Maybe a last suggestion: `_row_sum_of _squares` rather than `_sum_of _squares` better describes the function.